lib/ice_cube/validations/monthly_by_set_pos.rb

```diff
@@ -0,0 +1,85 @@
+module IceCube
+
+  module Validations::MonthlyBySetPos
+
+    def by_set_pos(*by_set_pos)
+      return by_set_pos([by_set_pos]) if by_set_pos.is_a?(Integer)
+
+      unless by_set_pos.nil? || by_set_pos.is_a?(Array)
+        raise ArgumentError, "Expecting Array or nil value for count, got #{by_set_pos.inspect}"
```

Comment:
This validation seems unnecessary as well. `by_set_pos` **is always** an `Array`.